content/en/mobile/_index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Datadog Oncall Mobile App
+kind: documentation
+beta: true
+further_reading:
+- link: "/monitors/"
+  tag: "Documentation"
+  text: "Alerting"
+- link: "/dashboards/"
+  tag: "Documentation"
+  text: "Dashboards"
+---
+
+<div class="alert alert-warning"> This service is in public beta. To join the beta, follow the installation instructions. The beta is not currently available to customers using SAML authentication. If you have any feedback, please <a href="/help">contact Datadog support</a>.</div>monitors
+
+The Datadog Oncall app enables on-call engineers to view alerts from Datadog on their mobile device. When notified to a Datadog alert by Pagerduty, Slack or other pager apps, you'll be able to investigate issues by opening monitor graphs and dashboards on your mobile device.
+
+## Installing
+
+The Datadog Oncall App is currently in beta, so it's not available generally in app stores. To download the app, use the specific invite links in these instructions.
+
+### iOS
+
+1. Install the [Testflight app][1] from the App Store.
+2. Open this invite link from your iPhone: [https://apple.co/2xEC5ke][2]. Allow the link to open in the Testflight app. This takes you to a page where you can download the Datadog app.
+3. Once the app downloads, open the app, select whether you are logging into the US or EU site, and log in with your Datadog credentials.
+
+### Android
+
+1. To install the android app, open this invite link from your android: [https://play.google.com/apps/testing/com.datadog.app][3]. This takes you to a page where you can download the Datadog app.
+2. Once the app downloads, open the app, select whether you are logging into the US or EU site, and log in with your Datadog credentials.
+
+### Logging in
+
+You can login using standard and Google authentication. [SAML login][4] is not currently supported. If you have SAML enabled but not strictly, you can access the mobile app by creating a new standard email account. An org admin needs to approve the newly created account.
+
+Switch organizations and log out from the **Account page**.
+
+## Monitors
+
+{{< img src="mobile/monitors.jpg" alt="monitor page" style="width:50%;">}}
+
+On the Monitors page, you can view and search all of the monitors that you have access to in your Datadog org. You can specify by field name and build specific search queries based on your tagging strategy. For more information about searching, see the [Manage Monitors Search section][5]. For example, to filter only metric monitor related to SRE team that are alerting, use the query `"status:Alert type:Metric team:sre"`. Click into individual alerts to see details, which can be filtered by type and by alert time. You can also mute the alert. Your ten most recent searches are saved so that you can faster access previous queries.
```

Comment:
```suggestion
On the Monitors page, you can view and search all of the monitors that you have access to in your Datadog org. You can specify by field name and build-specific search queries based on your tagging strategy. For more information about search, see the [Manage Monitors Search section][5]. For example, to filter on metric monitors related to the SRE team that is being alerted, use the query `"status:Alert type:Metric team:sre"`. Click into individual alerts to see details, which can be filtered by type and by alert time. You can also mute the alert. Your ten most recent searches are saved so that you have faster access previous queries.
```